cmd/server/command/start.go

```diff
@@ -249,37 +280,65 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 						"commit-link", opts.Spec.Application.URL,
 						"commit-sha", opts.Spec.Application.SHA,
 						"releaser", opts.Releaser,
-						"type", "release")
-					logger.Infof("Release [%s]: %s (%s) by %s, author %s", opts.Environment, opts.Service, opts.Spec.ID, opts.Releaser, opts.Spec.Application.AuthorName)
+						"type", "release",
+					)
+					logger.Infof(
+						"Release [%s]: %s (%s) by %s, author %s",
+						opts.Environment,
+						opts.Service,
+						opts.Spec.ID,
+						opts.Releaser,
+						opts.Spec.Application.AuthorName,
+					)
 				},
-				"prometheus": func(ctx context.Context, opts flow.NotifyReleaseOptions) {
-					metricsObserver.ObserveRelease(metrics.Release{
-						Environment: opts.Environment,
+				"prometheus": func(_ context.Context, opts flow.NotifyReleaseOptions) {
+					metricsObserver.ObserveRelease(
+						metrics.Release{
+							Environment: opts.Environment,
+							Service:     opts.Service,
+							Releaser:    opts.Releaser,
+							Intent:      opts.Intent.Type,
+						},
+					)
+				},
+				"events": func(ctx context.Context, opts flow.NotifyReleaseOptions) {
```

Comment:
@kjuulh Looks good to me - Thanks for spending time on it!

I have some questions:)
It means that the queue defaults to "release-manager" - do you know if this is true? And do you know where the value is retrieved from on start https://github.com/lunarway/release-manager/blob/master/cmd/server/command/root.go#L129?

release-manager uses the github.com/rabbitmq/amqp091-go directly. 
When we need to consume, would be possible to go through go.lunarway.com/amqp and what approach would you recommend?

Reply:
We cannot go through go.lunarway.com/amqp as that is a private library. You would need to listen on the exchange in the same manner as we do in release manager it self

Reply:
No, but we can in development-metrics :) Alright.

Reply:
I will add poderror event when we've gotten your subscription stuff to work